### DISCUSSION TITLE
Ability to arrange Cruds

### DISCUSSION THREAD

**Discussion by silicahd** [TiagoSilvaPereira/vemto-issues#274] [Ideas & Suggestions]
Would be nice to arrange these Cruds. They are all over the place right now. Maybe like modules. 

![image](https://user-images.githubusercontent.com/7761519/153529518-6365d6ec-f6ba-42e7-87f5-ab09e59ba184.png)

**Comment by jgulledge19** [comment]
That would be a good feature! Maybe alphabetically to start with and then allow the user to drag them in order as desired. Also this could related to #292 to group them by namespace.
